egs2/librispeech/discrete_asr1/local/data.sh

```diff
@@ -0,0 +1,452 @@
+#!/usr/bin/env bash
```

Comment:
I'm not sure making the whole data preprocessing to local is a good choice. My concern is that the method might need more extra efforts when applying to other datasets. For the first version, I think it might be good, but for generalization ability, at least we could organize some stages (kmeans etc.) to the pyscripts/scripts. My preference is still to put into TEMPLATE though, as that's the reason we hold the template for each task.

Reply:
I thought most of the functions are similar to hubert-pretraining one, can we reuse that?

Reply:
Indeed. Now I reused kmeans training and label dumping from hubert-training.